docs/general/dql/selects.rst

```diff
@@ -511,274 +502,461 @@ does always return ``NULL`` when comparing ``NULL``.
     +----------+
     SELECT 1 row in set (... sec)
 
-.. _sql_dql_any_array:
+.. _sql_dql_arrays:
 
-``ANY (array)``
-===============
+Arrays
```

Comment:
The sections before were about comparison operators, how does a general `Arrays` section fit here?

Reply:
good point. I moved it because these sections concern operations on arrays, so I figured they should come after that info. but you're right that this info should be a part of the general comparison operators

to solve both issues, I moved the sections up (as suggested) and added a ref to the opening paras of both sections that links to the arrays section so people can jump there if they need to

fix in https://github.com/crate/crate/pull/10866/commits/4ce0d3dbe7933a5e54466abaab23f806bd6da9ee